idd/Energy+.idd.in

```diff
@@ -40191,6 +40191,10 @@ ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
         \reference ZoneTerminalUnitNames
         \reference DOAToZonalUnit
         \reference ZoneEquipmentNames
+        \reference-class-name validBranchEquipmentTypes
+        \reference validBranchEquipmentNames
+        \reference-class-name validOASysEquipmentTypes
+        \reference validOASysEquipmentNames
```

Comment:
TU now allowed on main branch and in OA system

Reply:
Update the object \memo to mention the new applications.